modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -417,6 +417,9 @@ public static class Storage {
 
         /** Error reading from an index that has not yet been built. */
         public static final int INDEX_NOT_BUILT_ERR = STORAGE_ERR_GROUP.registerErrorCode((short) 1);
+
+        /** Error indicating a fatal error in the storage layer. */
```

Comment:
I wouldn't use the term "**layer**" in the project, it's not often used, especially in storage modules, it's up to you.

The error description is more general and the code is more specific. Let's fix the description and make it more detailed, something like: Error detecting corrupted data.